src/main/java/seedu/duke/profile/Profile.java

```diff
@@ -131,7 +136,7 @@ private static void checkBmiValidity(int bmi) throws InvalidCharacteristicExcept
      **/
     public static String retrieveBmiStatus(double bmi) {
         String result;
-        assert bmi > 0;
+        assert bmi > 0 : "BMI Cannot be non-positive.";
```

Comment:
I think there is a typo here? Should it be `BMI cannot be non-positive" (C with lowercase) ?